src/parseTree/nodes/asmt.java

```diff
@@ -1,23 +1,21 @@
 package src.parseTree.nodes;
 
-import src.parseTree.categories.type;
-import src.parseTree.tokens.asmt_op;
-import src.parseTree.tokens.end_stmt;
-import src.parseTree.tokens.id;
+import java.util.ArrayList;
+import java.util.List;
 
-public class asmt extends stmt<Integer> {
-    private type t;
-    private id name;
-    private asmt_op op;
-    private expr exp;
-    private end_stmt endStmt;
```

Comment:
Why did you remove the categories for ONLY some of the nodes and not all?

Reply:
Must be returned to retain compatibility for execution

Reply:
I tried not to remove logic, since it wasnt necessary for parser. That being said, there was no reason to remove the fields and constructor. At the time, I was thinking the fields were unnecessary assuming the children are in the correct order (which is true if parser works correctly), but having a `fix` method and then using the fields would be more readable.